spec/requests/api/v1/submissions_swagger_spec.rb

```diff
@@ -109,19 +109,40 @@
         security [{ oAuth: [] }]
         parameter name: :id, in: :path, type: :string
 
-        response '200', 'submission found' do
-          let(:expected_response) do
-            {
-              submission: id,
-              status: 'processing',
-              _links: [
-                href: "http://www.example.com/api/v1/submission/status/#{id}"
-              ]
-            }
+        context 'when the submission has not completed' do
+          response '202', 'incomplete submission found' do
```

Comment:
I know you haven't actually changed it in this PR (just re-namespaced it) but could this description be clearer? On its own it seems fine, but on the swagger page, it appears as... 
![image](https://user-images.githubusercontent.com/6757677/134358175-96419d27-7a1f-41be-bf87-e4c47e07f250.png)

Would `Submission still processing` (or something) be easier to understand as a novice user?

Reply:
Fair point. I'll update when I get back from the school run 🏃‍♂️

Reply:
Or we could have a group review of Swagger as a whole when the current batch of tickets are done?